packages/terra-site/src/cerner-mock-theme.scss

```diff
@@ -3,6 +3,10 @@
 @import '~terra-mixins';
 
```

Comment:
The `themeable-variables.json` file needs to be updated. See this issue: https://github.com/cerner/terra-core/issues/1106.